about.html

```diff
@@ -5,72 +5,78 @@
-                    </div>
-                </nav>
-            </div>
-        </section>
+                    </span>
+                </div>
+            </section>
+
+            <section>
+                <div class="nav-container">
+                    <img src="/assets/img/logos/logo-icon/logo-icon-white.svg" alt="Who Travels Logo" class="logo">
+                    <nav class="nav-menu">
+                        <ul class="menu">
+                            <li class="nav-li"><a href="book.html" class="top-nav" id="book">Book Trip</a></li>
+                            <li class="nav-li"><a href="destinations.html" class="top-nav" id="destinations">Destinations</a></li>
+                            <li class="nav-li"><a href="about.html" class="top-nav" id="about">About Us</a></li>
+                            <li class="nav-li"><a href="contact.html" class="top-nav" id="contact">Contact</a></li>
+                            <li class="nav-li"><a href="explore.html" class="top-nav" id="explore">Explore</a></li>
+                        </ul>
+                        <div class="hamburger-icon">
+                            <span class="hamburger"></span>
+                            <span class="hamburger"></span>
+                            <span class="hamburger"></span>
+                        </div>
+                    </nav>
+                </div>
+            </section>
+        </div>
     </header>
+
     <!-- End Header -->
 
 
     <section class="about-grid">
         <div class="about-text">
 
-            <p> Established on Earth in the year 2021, Who Travels started out very small, with only trips to past and present Earth locations and Mars. Now with access to nearly every Earth era and over 60% of the Galaxy, Who Travels is the largest (and only!) Time and Space Travel Company on any planet!</p>
+            <p>
+                Established on Earth in the year 2021, <span class="shoutout">Who Travels</span> started small, with only trips to past and present Earth locations and Mars. Now with access to nearly every Earth era and over 60% of the Galaxy, <span class="shoutout">Who Travels</span> is the largest (and only!) Time and Space Travel Company on any planet!
+            </p>
 
-            <p>Who Travels offers instantaneous trips that will have you home before breakfast or overnight transit with comfortable accommodations. Overnight excursions include meals from our gourmet food machine, spacious lounging rooms, a swimming pool, a conservatory, and a personal view from the control room. (rotating image of interior?)</p>
+            <p><span class="shoutout">Who Travels</span> offers instantaneous trips that will have you home before breakfast or overnight transit with comfortable accommodations. Overnight excursions include meals from our gourmet food machine, spacious lounging rooms, a swimming pool, a conservatory, and a personal view from the control room. (rotating image of interior?)</p>
```

Comment:
I don't know how I feel with the styling of the "shoutout" section -- I think just having it bolded would work and look a bit less clunky.

Reply:
I think I agree, I changed it to bold